src/lib/components/Line.svelte

```diff
@@ -27,6 +27,26 @@
 		isSpinning = false;
 		URL.revokeObjectURL(urlObject);
 	}
+
+	import {
+		getDoneStatus,
+		doneStatusPage,
+		getDoneStatusPage,
+		toggleDone as toggleDoneFile
+	} from '$lib/todo-file';
+	let isDone = false;
+	$: {
+		isDone = $doneStatusPage ? getDoneStatus($page.url + '/' + data.name) : false;
+	}
```

Comment:
Perché non lo collassiamo in questo?

```suggestion
	$: isDone = $doneStatusPage ? getDoneStatus($page.url + '/' + data.name) : false;
```